### PR TITLE
fix: TIDDIT VCF header

### PR DESCRIPTION
**Pull request by khurrammaqbool**
### This PR:

Fixed: VCF header for SVs.

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Review by mathiasbio — APPROVED** [review]
Nice and quick, thanks for fixing this! 🧙 
It has been tested with success and I have no comments, so I'm approving this 👍

**Comment by khurrammaqbool** [comment]
Screenshot 2023-07-21 at 10 00 18

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,11 @@
+[X.X.X]
```
Comment (ivadym) [inline CHANGELOG.rst:1]:
```suggestion
[12.0.2]
```

**Review by ivadym — COMMENTED** [review]
👍 

We can't proceed with the merge until the pytest issue is resolved. Please take a look at the pull requests that require review @Clinical-Genomics/cancer-bioinfo:
https://github.com/orgs/Clinical-Genomics/projects/46/views/3

**Comment by ivadym** [comment]
I have created a branch for release v12.0.2 and rebased the pull requests we intend to include. Please address any conflicts and and squash and merge them into that branch when ready 🥳

**Pull request merged**